### PR TITLE
Fix unicode decode error

### PR DESCRIPTION
**Pull request by IvanMalison**

### CODE REVIEW

File: .dir-locals.el [hunk 1]
```diff
@@ -0,0 +1 @@
+((nil . ((pytest-global-name . "tox -e py27 --"))))
```
Comment (hugovk) [inline .dir-locals.el]: What's this .dir-locals.el file for?

**Comment by hugovk** [comment]
Python 3.2, 3.3 and 3.4:

 * tests/request_test.py::test_get_cache_key[artist2] FAILED

e.g. https://travis-ci.org/pylast/pylast/jobs/46293471

Passed for others.

File: .dir-locals.el (hunk 1 above)
Comment (IvanMalison) [inline .dir-locals.el]: Its for emacs. Sorry, I meant to ask if it is okay to add it to the repository.

**Comment by IvanMalison** [comment]
Yeah, this is broken.  I might try to get vcrpy wroking so that its easier to run test locally.

**Comment by IvanMalison** [comment]
Justification for the replacement of _string with a new definition:
https://github.com/IvanMalison/_string_unicode_testing_pylast

**Comment by IvanMalison** [comment]
Finally got a totally clean build https://travis-ci.org/IvanMalison/pylast/builds/46382232 . I think the massive decrease in coverage reported by coveralls is because all of the tests get disabled when running for a PR...

File: clonedigger.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+clonedigger pylast
+! grep -E "Clones detected\|lines are duplicates" output.html
```
Comment (hugovk) [inline clonedigger.sh]: Ignoring the ! for now, this doesn't work when clones are detected (and there are right now). A simple test on Travis:
```
$ echo "Clones detected" | grep -E "Clones detected\|lines are duplicates"
$ echo "Clones detected" | grep -E "Clones detected|lines are duplicates"
Clones detected
$ echo "Clones detected" | grep -e "Clones detected\|lines are duplicates"
Clones detected
$ echo "Clones detected" | grep -e "Clones detected|lines are duplicates"
```

**Comment by hugovk** [comment]
Thank you!

**Pull request merged**